scripts/Units/armsnipe.bos

```diff
@@ -7,7 +7,7 @@ piece  flare, torso, head, pelvis, rthigh, rleg, lleg, lthigh,
        ruparm, luparm, lloarm, lhand, rloarm, gun1, gun2,
        barrel1, barrel2, laser, rhand, rfoot, lfoot, aimy1;
 
-static-var  GunOpen, bMoving, Static_Var_3, Static_Var_4, restore_delay, wpn1_lasthead, animSpeed, maxSpeed,
+static-var  GunOpen, bMoving, Static_Var_3, Static_Var_4, restore_delay, laser_hide, wpn1_lasthead, animSpeed, maxSpeed,
```

Comment:
```suggestion
static-var  GunOpen, bMoving, Static_Var_3, Static_Var_4, restore_delay, reholster_delay, wpn1_lasthead, animSpeed, maxSpeed,
```
`laser_hide` sounds like it has to do with showing/hiding model pieces. Does `reholster_delay` capture the semantic meaning of this var?

Reply:
yes, but the reholster is much more related to restore_delay. "laser_persist" might be a better name, because this determines time something is visible after stopping aim.(used to be magic number)